### ISSUE TITLE
Bitvavo csv header changed

### ISSUE THREAD

**Issue by bart268** [dickwolff/Export-To-Ghostfolio#94]
For me Bitvavo import isn't working, it looks like the csv header changed a little.

Old CSV header (according to the sampe csv in this git)
Timezone,Date,Time,Type,Currency,Amount,Price (EUR),EUR received / paid,Fee currency,Fee amount,Status,Transaction ID,Address

New CSV header I got from Bitvavo export
Timezone,Date,Time,Type,Currency,Amount,Quote Currency,Quote Price,Received / Paid Currency,Received / Paid Amount,Fee currency,Fee amount,Status,Transaction ID,Address

Extra fields added:
Quote Currency,Quote Price

So this gives the following error:
Error: An error ocurred while parsing! Details: Invalid Record Length: columns length is 13, got 15 on line 2

To make the import work again I removed the two extra fields.

**Comment by dickwolff** [comment]
Hi there! Sorry for the late response as I've been busy.

I downloaded a new Bitvavo export myself to verify and it had the same format as you specified. I have changed the format to account for this. There is a new [beta image](https://hub.docker.com/layers/dickwolff/export-to-ghostfolio/0.16.1-beta/images/sha256-88b8f2f21465d38d6c25aa9c84a6815b78f60d6a8e7ee46a36ea9dc0a07ab53f?context=explore) available now for you to try.
